index.js

```diff
@@ -1,9 +1,16 @@
 'use-strict';
 
-var xml = require('xml');
-var Base = require('mocha').reporters.Base;
-var filePath = process.env.MOCHA_FILE || 'test-results.xml';
-var fs = require('fs');
+var fs      = require('fs');
```

Comment:
I would prefer to keep the original whitespace in these statements as they were for consistency.